src/app/conf/2025/layout.tsx

```diff
@@ -52,9 +52,7 @@ export default function Layout({
           { children: "FAQ", href: "/conf/2025/#faq" },
         ]}
       />
-      <ThemeProvider attribute="class">
```

Comment:
This means the old conference websites are now inside of `ThemeProvider`, right?

We should look through them. I don't think they relied on `.dark` class, but I'll spend a few minutes once the deploy preview is up.

Reply:
Ok, i certainly seem to have bungled this bit...I had a feeling there was something about these different theme strategies I was missing. That said, take a look at [the live conf/2024](https://graphql.org/conf/2024/)...this is broken already.